temporian/core/operators/test/test_arithmetic_multi_index.py

```diff
@@ -0,0 +1,135 @@
+# Copyright 2021 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+from absl.testing import absltest
+
+from temporian.implementation.numpy.data.io import event_set
+
+from temporian.test.utils import f32, f64, assertOperatorResult
+
+# store ids
+TRYOLABS_SHOP = 42
+GOOGLE_SHOP = 101
+# product ids
+MATE_ID = 1
+BOOK_ID = 2
+PIXEL_ID = 3
```

Comment:
i'd substitute these for just `1` and `2` in the tests themselves. no need to add complexity in the tests.